src/components/BottomNav.jsx

```diff
@@ -0,0 +1,32 @@
+import { ReactComponent as AddIcon } from 'assets/add.svg';
+import FilterButtons from 'components/FilterButtons';
+import Button from 'components/ui/Button';
+import { useDispatch } from 'react-redux';
+import { setIsAddingTask } from 'redux/actions/TodoAction';
+
+function BottomNav() {
+  const dispatch = useDispatch();
+
+  function setIsAdding(event) {
+    event.preventDefault();
+    dispatch(setIsAddingTask(true));
+  }
+
+  return (
+    <div>
```

Comment:
Unnecessary div. If we don't need it, can we replace it with `React.Fragment` ?

Reply:
Replaced with fragment. thanks